src/geocat/f2py/missing_values.py

```diff
@@ -1,42 +1,52 @@
 import numpy as np
 
-#all missing values are represented by all 1's in their dtype, int8 would be bin_11111111 or dec_-128
+# all missing values are represented by the maximum value in their dtype,
```

Comment:
Is it intentional to change the use of minimum to maximum? I'd expect using min to be better safe than max (requiring the sign-bit to be 1 in addition to the rest of the bits) for missing value representation.

Reply:
I made that change for the following reasons:
A) The bitwise format of floating point numbers has no condition in which all the bits are 1s.
B) The unsigned numbers already had to be max, as min would be 0, which is not a good option when trying to avoid collisions of msg and real data.
C) The min value for int8 is -128, which is bin_10000000 NOT bin_11111111 (which is -1) signed negative ints use two's compliment representation. 
D) Its simpler for a user to understand that a max range value is an error. 

Reason D was my primary reason, and I chose it after I realized that our previous explanation of "all the bits are 1s" was incorrect. 

note: In theory fortran *should* be able to handle `np.nan` as the bitwise representation is standard per IEEE-754, but I don't want to risk breaking things and needing to fix them when we are about to replace them anyway. 

So this seemed like an easy simplification to make, and it overall is a minor change, as no end user is going to notice the difference between max and min in their use cases.

Reply:
Okay, thanks for the clarification!